tests/api/v1/auditEvents/post.js

```diff
@@ -41,6 +41,18 @@ describe('tests/api/v1/auditEvents/post.js >', () => {
     .end(done);
   });
 
+  /*
+   * Swagger converts the object to an array if swagger.yaml expects the
+   * schema type of the request payload to be an array.
+   */
```

Comment:
@iamigo Added the test that we talked about but you might want to have a look at the comments above.

Reply:
OK